xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">172.23.44.176</domain>
+    </domain-config>
+</network-security-config>
```

Comment:
Is it a duplication with the ```res/xml/network_security_config.xml``` file? I don't know ... If both files are needed please ignore me.